examples/stcc/stcc.cpp

```diff
@@ -139,9 +139,15 @@ ExprPtr biorthogonal_transform(
   // Coefficients
   std::vector<double> bt_coeff_vec;
   {
+    auto factorial = [](auto x) {
+      if (x > 20)
+        throw std::runtime_error("std::intmax_t running out of precision");
+      return boost::numeric_cast<std::intmax_t>(sequant::factorial(x));
+    };
```

Comment:
Wouldn't it make more sense to use `std::uintmax_t` instead of `std::intmax_t`?

Reply:
will address via https://github.com/ValeevGroup/SeQuant/issues/110